content/usecaseItems.md

```diff
@@ -1,5 +1,5 @@
 ---
 example_usecase_id: 0
-example_usecase_name: Example usecase
-example_usecase_description: This is an example usecase.
+example_usecase_name: Multi-year prediction of drought and heat stress to support decision making in the wheat sector
```

Comment:
can we summarize the name? something like: `Multi-year prediction of drought and heat stress in the wheat sector`

Reply:
Fixed.